### ISSUE TITLE
Tooltip plugin switch

### ISSUE THREAD

**Issue by martgnz** [PopulateTools/gobierto#767]
[Tipsy](https://github.com/jaz303/tipsy) had its last update five years ago. While for simple use cases works fine, it doesn't support simple things like triggering the tooltip. The positioning doesn't work well for some cases (for fixed positions and on phones as well), and sometimes it feels hacky (once I had to manually add stuff which wasn't supported on the code).

Maybe we could switch to a simpler, well maintained library. A familiar option is using [Bootstrap's plugin](http://v4-alpha.getbootstrap.com/components/tooltips/), which is heavily inspired by Tipsy but is actively maintained and uses modern css3. The caveat is that it relies on another library for the positioning.

[Tippy.js](https://github.com/atomiks/tippyjs) seems good too, with a lot of options.

TL;DR: Tipsy is not maintained. We can keep using it, but has weird edge cases. Should we switch?
